### PR TITLE
refactor: split out packages and clean up deps

### PR DESCRIPTION
**Pull request by craigbilner**
* feat: split out markup traversal from presentation - move article renderers into article (ad, key-facts, pull-quote)
* feat: create provider-queries package for non-circular sharing
* test: remove jest-configurator dependency from test-utils
* chore: update cli
* chore: bump versions to remove CLI noise (styled-components have made a change to the way they print their classes)
* chore: remove unnecessary svg dependency from serializer test (will revisit tests/coverage)

### CODE REVIEW

File: packages/markup-forest/README.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
+# Markup Forest
```
Comment (colincclark) [inline packages/markup-forest/README.md]: Weird name. What about calling this one `markup-handler` and the other one `markup-core`, or similar?

File: packages/markup/__tests__/shared.base.js [hunk 2]
```diff
@@ -1,229 +1,84 @@
...
-const lineBreak = require("../fixtures/line-break.json");
-const mixture = require("../fixtures/tag-mixture.json");
-const nested = require("../fixtures/nested.json");
-const bio = require("../fixtures/bio.json");
-const script = require("../fixtures/script.json");
-const image = require("../fixtures/image.json");
+import renderTrees, { renderTree } from "@times-components/markup-forest";
+import coreRenderers from "../src/markup";
+import paragraph from "../fixtures/paragraph.json";
+import block from "../fixtures/block.json";
```
Comment (colincclark) [inline packages/markup/__tests__/shared.base.js]: `blockAST` to be explicit, etc?

Comment (colincclark) [reply packages/markup/__tests__/shared.base.js]: or is it worth refactoring the `renderTrees` API to take an object `{ast: blah, renderers: blah }`

Comment (craigbilner) [reply packages/markup/__tests__/shared.base.js]: I can call it `blockAST` if it's helpful

File: packages/markup-forest/README.md (hunk 1 above)
Comment (craigbilner) [reply packages/markup-forest/README.md]: it''s sort of a pun, it exposes `renderTrees` so plural of `tree` is a forest..... :evergreen_tree: :deciduous_tree:

File: packages/markup-forest/__tests__/__snapshots__/markup-forest.test.js.snap [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+    <special>
+      special: {"name":"specialElement","attributes":{},"children":[{"name":"text","attributes":{"value":"text of a special element"},"children":[]},{"name":"inline","attributes":{},"children":[{"name":"text","attributes":{"value":"inline text of a special element"},"children":[]}]}]}
+    </special>
+     after it too
+  </div>
+   and here
+</div>
+`;
+
+exports[`6. provide ast of node for child 1`] = `null`;
```
[the remark this answers is not in the document]
Comment (craigbilner) [reply packages/markup-forest/__tests__/__snapshots__/markup-forest.test.js.snap]: yep, will fix :+1:

**Pull request merged**